docs/rest_api.rst

```diff
@@ -71,862 +74,584 @@ Following are the available API endpoints with which you can interact with Raide
 Querying Information About Your Raiden Node
 ===============================================
 
-Querying your address
----------------------
+.. http:get:: /api/(version)/address
 
-When raiden starts you choose an ethereum address which will also be your raiden address. You can query that address by making a ``GET`` request to the ``/api/<version>/address`` endpoint.
+   Query your address. When raiden starts, you choose an ethereum address which will also be your raiden address.
 
-Example Request
-^^^^^^^^^^^^^^^
+   **Example Request**:
 
-``GET /api/1/address``
+   .. http:example:: curl wget httpie python-requests
 
-Example Response
-^^^^^^^^^^^^^^^^
-``200 OK`` and
+      GET /api/1/address HTTP/1.1
+      Host: localhost:5001
 
-::
+   **Example Response**:
+
+   .. sourcecode:: http
 
-    {"our_address": "0x2a65aca4d5fc5b5c859090a6c34d164135398226"}
+      HTTP/1.1 200 OK
+      Content-Type: application/json
+
+      {
+          "our_address": "0x2a65Aca4D5fC5B5C859090a6c34d164135398226"
+      }
 
 Deploying
 =========
 
-Registering a Token
--------------------
+.. http:put:: /api/(version)/tokens/(token_address)
+
+   Registers a token. If a token is not registered yet (i.e.: A channel manager for that token does not exist in the network), we need to register it by deploying a channel manager contract for that token.
 
-If a Token is not registered (i.e.: A channel manager for that token does not exist in the network) then we need to register it by deploying a Channel Manager contract for that token.
+   **Example Request**:
 
-We can do that by doing a ``PUT`` request to the endpoint ``/api/<version>/tokens/<token_address>``. The request should return the deployed channel manager's address.
+   .. http:example:: curl wget httpie python-requests
 
-If the token is already registered then a 409 Conflict error is returned.
+      PUT /api/1/tokens/0xEA674fdDe714fd979de3EdF0F56AA9716B898ec8 HTTP/1.1
+      Host: localhost:5001
 
-Example Request
-^^^^^^^^^^^^^^^
+   **Example Response**:
 
-``PUT /api/1/tokens/0xea674fdde714fd979de3edf0f56aa9716b898ec8``
+   .. sourcecode:: http
 
-Example Response
-^^^^^^^^^^^^^^^^
-+------------------+----------------------------+
-| HTTP Code        | Condition                  |
-+==================+============================+
-| 201 Created      | For successful Registration|
-+------------------+----------------------------+
-| 409 Conflict     | If the token is already    |
-|                  | registered                 |
-+------------------+----------------------------+
+      HTTP/1.1 201 CREATED
+      Content-Type: application/json
 
-And with a 201 response we also get the address as can be seen below::
+      {
+          "channel_manager_address": "0xC4F8393fb7971E8B299bC1b302F85BfFB3a1275a"
+      }
 
-    {"channel_manager_address": "0xC4F8393fb7971E8B299bC1b302F85BfFB3a1275a"}
+   :statuscode 201: A channel manager for the token has been successfully created.
```

Comment:
I kinda liked how the tables you are removing were rendered. How does this render in read the docs? Do you have an example rendering somewhere to see?

Reply:
This is what it will look like:

![bildschirmfoto von 2018-06-29 17-24-43](https://user-images.githubusercontent.com/11137424/42100853-a4cc9072-7bc1-11e8-9a27-4802cd6ba4ff.png)